__tests__/services/usersServices/getSingleUser.test.js

```diff
@@ -0,0 +1,38 @@
+const User = require("../../../models/user");
+const { getSingleUser } = require("../../../api/users/services");
+
+jest.mock("../../../models/user");
+
+const id = "123";
+
+describe("getSingleUser function", () => {
+  it("should return user, when user exists", async () => {
+    User.findOne.mockResolvedValueOnce(() => ({
```

Comment:
Why in this test you use `mockResolvedValueOnce` but in the next one `mockImplementationOnce`.